### PR TITLE
Fixed Rey Alias and add a few other new ones

### PR DESCRIPTION
**Pull request by UNOMAHANLYON**

### CODE REVIEW

File: units.go [hunk 1]
```diff
@@ -85,10 +89,14 @@ func CharName(src string) string {
 		return "CT-5555 \"Fives\""
 	case "rex", "zrex":
 		return "CT-7567 \"Rex\""
+	case "Malak":
```
Comment (ronoaldo) [inline units.go]: Since we lowercase before comparing in the switch/case, we need to check always to lowercase too.

**Review by ronoaldo — COMMENTED** [review]
Would you like me to fix the comments after merge or want to re-submit the PR?

File: units.go (hunk 1 above)
Comment (UNOMAHANLYON) [reply units.go]: fixed

File: units.go [hunk 2]
```diff
@@ -85,10 +87,14 @@ func CharName(src string) string {
...
+		return "Darth Malak"
 	case "maul", "zaul", "zmaul", "darth maul":
 		return "Darth Maul"
 	case "dn", "nihilus", "znihilus", "darth nihilis":
 		return "Darth Nihilus"
+	case "DR":
```
Comment (UNOMAHANLYON) [inline units.go]: fixed this one too

File: units.go [hunk 3]
```diff
@@ -285,6 +289,8 @@ func CharName(src string) string {
 		return "Stormtrooper Han"
 	case "sf":
 		return "Sun Fac"
+	case "SLKR", "SLK":
```
Comment (UNOMAHANLYON) [inline units.go]: and this one

**Comment by ronoaldo** [comment]
LGTM! Thanks, merging in!

**Pull request merged**